src/types/url.types.ts

```diff
@@ -0,0 +1,15 @@
+export interface UrlType {
+    Id: number;
+    OriginalUrl: string;
+    ShortUrl: string;
+    Comment: string;
+    UserId: number;
```

Comment:
UserId is number or it can be alphanumeric ?

Reply:
![image](https://github.com/Real-Dev-Squad/tiny-site-frontend/assets/70854507/cc4ed4ac-e4b2-420e-b3f2-9fd20b373b28)